docs/docs/modules/agents/agent_types/plan_and_execute.mdx

```diff
@@ -1,5 +1,9 @@
 # Plan and execute
 
+:::info Compatibility
```

Comment:
Is this true?

Reply:
Lemme double check, that change is just me reordering where it was on the page to match simliar docs

Reply:
Yes, looks like it is. It's calling `ChatAgent` a few times inside the code